src/tests/momentOfTruthTest.ts

```diff
@@ -51,9 +53,13 @@ class MomentOfTruthTest {
     let stub4 = sinon.stub(devOps, "createPullRequestProperties").returns({
       workingDir: cwd + "/src/tests/Resource/momentOfTruth/old",
       checkout: () => "true",
-      diff: () => {
+      diff: async () => {
         return [{ path: "specification/test-lint/test/test-lint-result.json" }];
```

Comment:
Why does this constant value return need to be async?

Reply:
removed it